### PR TITLE
mavenangulardeduplication

### PR DESCRIPTION
**Pull request by lawrence-lee**

### CODE REVIEW

File: maven/apps/npm-deduplication-portlets/angular/angular-provider/bnd.bnd [hunk 1]
```diff
@@ -0,0 +1,10 @@
+Bundle-Name: Angular Provider
+Bundle-SymbolicName: com.liferay.blade.angular.provider
+Bundle-Version: 1.0.0
+-contract: JavaPortlet
```
Comment (lawrence-lee) [inline maven/apps/npm-deduplication-portlets/angular/angular-provider/bnd.bnd:4]: @gamerson I'm not sure we need to add these plugins for this provider module. Can you remove the ones we dont need?

**Comment by gamerson** [comment]
@lawrence-lee  do we have any functional tests that cover these?

**Comment by lawrence-lee** [comment]
@gamerson Yes, in `liferay-workspace`

**Comment by gamerson** [comment]
ci:test:71

**Comment by gamerson** [comment]
Just started reviewing :) <br><br>:octocat: *Sent from [GH](http://nodegh.io).*

**Pull request merged**